content/en/docs/tasks/security/authorization/authz-tcp/index.md

```diff
@@ -55,7 +59,9 @@ Get the pod IP address and send the request with the following command:
 
     {{< text bash >}}
     $ TCP_ECHO_IP=$(kubectl get pod "$(kubectl get pod -l app=tcp-echo -n foo -o jsonpath={.items..metadata.name})" -n foo -o jsonpath="{.status.podIP}")
-    $ kubectl exec "$(kubectl get pod -l app=sleep -n foo -o jsonpath={.items..metadata.name})" -c sleep -n foo -- sh -c "echo \"port 9002\" | nc $TCP_ECHO_IP 9002" | grep "hello" && echo 'connection succeeded' || echo 'connection rejected'
+    $ kubectl exec "$(kubectl get pod -l app=sleep -n foo -o jsonpath={.items..metadata.name})" \
+        -c sleep -n foo -- sh -c \
+        "echo \"port 9002\" | nc $TCP_ECHO_IP 9002" | grep "hello" && echo 'connection succeeded' || echo 'connection rejected'
```

Comment:
I see you didn't change this, but why are some of these commands using double quote which require escaping the inner quotes `"echo \"port ...` while others are using single quote `'echo "port ...`? I think it would be good to use single quote consistently in all these commands.

Reply:
nvm, I see that the reason is because some are expanding an ENV var in the string.